internal/testdata/dlahr2test/main.go

```diff
@@ -0,0 +1,100 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This program generates test data for Dlahr2. Test cases are stored in
+// gzip-compressed JSON file testlapack/testdata/dlahr2data.json.gz which is
+// read during testing by testlapack/dlahr2.go.
+//
+// This program uses cgo to call Fortran version of DLAHR2. Therefore, matrices
+// passed to the Fortran routine are in column-major format but are written into
+// the output file in row-major format.
+package main
+
+import (
+	"compress/gzip"
+	"encoding/json"
+	"log"
+	"math/rand"
+	"os"
+
+	"github.com/gonum/lapack/internal/testdata/netlib"
+)
+
+type Dlahr2Test struct {
+	N, K, NB int
+	A        []float64
+
+	AWant   []float64
+	TWant   []float64
+	YWant   []float64
+	TauWant []float64
+}
+
+func main() {
+	file, err := os.Create("../../../testlapack/testdata/dlahr2data.json.gz")
```

Comment:
Two dots is, assuming "/" is not. Use `filepath.Join("..", "..", "..", "testlapack/testdata/dlahr2data.json.gz")` if we care.